bin/generate.rb

```diff
@@ -0,0 +1,41 @@
+require 'optparse'
```

Comment:
```suggestion
#!/usr/bin/env ruby
require 'optparse'
```

We would also likely drop the file name extension, and would set the executable bit.